tests/components/samsungtv/test_device_trigger.py

```diff
@@ -0,0 +1,154 @@
+"""The tests for samsung device triggers."""
+from unittest.mock import patch
+
+import pytest
+
+import homeassistant.components.automation as automation
+from homeassistant.components.device_automation import DeviceAutomationType
+from homeassistant.components.media_player import DOMAIN as MP_DOMAIN
+from homeassistant.components.samsungtv.const import DOMAIN
+from homeassistant.setup import async_setup_component
+
+from . import setup_samsungtv_entry
+from .test_media_player import MOCK_ENTRY_WS_WITH_MAC
+
+from tests.common import (
+    MockConfigEntry,
+    async_get_device_automations,
+    async_mock_service,
+    mock_device_registry,
+    mock_registry,
+)
+from tests.components.blueprint.conftest import stub_blueprint_populate  # noqa: F401
+
+
+@pytest.fixture
+def device_reg(hass):
```

Comment:
Please try to add type hints to all tests:
```suggestion
def device_reg(hass: HomeAssistant) -> None:
```